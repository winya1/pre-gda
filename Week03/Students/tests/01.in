2
Larry Page 5 5 5
Sergey Brin 5 5 5
