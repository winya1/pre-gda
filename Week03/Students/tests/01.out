Case #1: Page, Larry: Grades: [5, 5, 5] (avg: 5.0)
Case #2: Brin, Sergey: Grades: [5, 5, 5] (avg: 5.0)
